.circleci/config.yml

```diff
@@ -403,8 +403,11 @@ jobs:
                 pip3 install tox==3.24.0
                 pip3 install chardet
                 pip3 install iso8601
-                pip3 install jupyter-repo2docker
-                jupyter-repo2docker --no-run ./tests/fixtures/
+        - run:
+            name: pull base docker images
+            command: |
+                docker pull nvidia/cuda:10.0-runtime
+                docker pull python:3.6-buster
```

Comment:
Not a big deal, but we would ideally keep this at the latest stable python which is 3.9 currently.

Reply:
I'll file a ticket to move tests to 3.9